src/util/options.schema.json

```diff
@@ -1737,6 +1837,44 @@
               "type": "boolean",
               "default": true
             },
+            "narrow-reuse": {
+              "title": "solvers.td3.narrow-reuse",
+              "description": "Reuse value when switching from widening to narrowing phase. Avoids one unnecessary re-evaluation.",
+              "type": "boolean",
+              "default": true
+            },
+            "narrow-reuse-verify": {
+              "title": "solvers.td3.narrow-reuse-verify",
+              "description": "Instead of reusing value when switching from widening to narrowing phase, just verify that the reuse would be correct by re-evaluating.",
+              "type": "boolean",
+              "default": false
+            },
+            "restart": {
+              "title": "solvers.td3.restart",
+              "type": "object",
+              "properties": {
+                "wpoint": {
+                  "title": "solvers.td3.restart.wpoint",
+                  "type": "object",
+                  "properties": {
+                    "enabled": {
+                      "title": "solvers.td3.restart.wpoint.enabled",
+                      "description": "Restart wpoint to bot when (re-)detected. Allows incremental to avoid reusing and republishing imprecise local values due to globals (which get restarted).",
+                      "type": "boolean",
+                      "default": false
+                    },
+                    "once": {
+                      "title": "solvers.td3.restart.wpoint.once",
+                      "description": "Restart wpoint only on first detection. Useful for incremental loading.",
+                      "type": "boolean",
+                      "default": false
+                    }
+                  },
```

Comment:
At one GobCon we decided to keep the wpoint restarting (which is independent of incrementality) since it's a very local change just in `eval` and allows TD3 to have some behavior similar to SLR4 or something, where it helps with some nested loop stuff I think.

This wasn't used in project benchmarks and didn't end up in the paper, but there are some defs in the bench repo where it is enabled when comparing side restarting to eliminate multiple variables from a single comparison.